conf/routes

```diff
@@ -390,6 +390,7 @@ GET            /api/files/:id/blob
 POST           /api/files/:id/remove                                                    @api.Files.removeFile(id: UUID)
 POST           /api/files/bulkRemove                                                    @api.Files.bulkDeleteFiles()
 GET            /api/files/:id/metadata                                                  @api.Files.get(id: UUID)
+GET            /api/files/:id/getSections                                               @api.Files.getSections(id: UUID)
```

Comment:
Could we change the API route to `GET /api/files/:id/sections` to be consistent and to follow REST rules? Thank you.